handlers/messagebird/handler.go

```diff
@@ -62,14 +62,16 @@ var statusMapping = map[string]courier.MsgStatus{
 	"expired":         courier.MsgStatusFailed,
 }
 
-type ReceivedMessage struct {
-	ID              string   `json:"id"`
-	Recipient       string   `json:"recipient"`
-	Originator      string   `json:"originator"`
-	Body            string   `json:"body"`
-	CreatedDatetime string   `json:"createdDatetime"`
-	MediaURLs       []string `json:"mediaUrls"`
-	MMS             bool     `json:"mms"`
+type formMessage struct {
+	ID              string   `name:"id"`
+	MID             string   `name:"mid"`  //shortcode only
```

Comment:
doesn't look like you're using gofmt.. we're all using vscode set to use default go formatter

Reply:
ah I used to, it must be turned off. I'll fix.